README.md

```diff
@@ -141,7 +141,7 @@ compare against the true GNU coreutils version on the Linux-based tests first.
 |         | tail      | Output the last part of files                    |
 |         | tee       | Redirect output to multiple files or processes   |
 | &check; | test      | Check file types and compare values              |
-|         | timeout   | Run a command with a time limit                  |
+| &check; | timeout   | Run a command with a time limit                  |
 |         | touch     | Change file timestamps                           |
```

Comment:
Off by one... the check needs to go on `touch`, since that's what you're implementing.

Reply:
Good find. Updated in [074fae1](https://github.com/vlang/coreutils/pull/146/commits/074fae1911404b0a7616f7eba46e3de59013b159) .